### ISSUE TITLE
Arel SQLiteral error

### ISSUE THREAD

**Issue by jtomaslobo** [geokit/geokit-rails#67]
Hello, I would like to know what can I do to avoid this error

NameError: uninitialized constant Arel::SqlLiteral

Whenever I try to use 

Item.within(3.9, :origin => [37.792, -122.393])

or anything similar.

Thanks,
Thomas

**Comment by dmastylo** [comment]
Anyone figured this out?

**Comment by jtomaslobo** [comment]
Not me, I am using a different lib right now

Enviado desde mi iPhone

**Comment by tagliala** [comment]
(partially?) Fixed via #65 

use the master branch
```rb
gem 'geokit-rails', github: 'geokit/geokit-rails'
```

You should also add units (e.g.: `units: :kms`): when I do not specify that parameter I got this error:

```
Mysql2::Error: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')
```

**Comment by esbanarango** [comment]
:+1:

**Comment by dmastylo** [comment]
Yeah I switched to track the master branch immediately after posting the issue, but the gem really should be pushed up so people installing it won't come up with the same issue!

@jtomaslobo what library are you using?

**Comment by mnoack** [comment]
@tagliala @esbanarango @dmastylo If someone will do a PR to test and fix the units issue (It would be nice to have an error like "you must set units", or perhaps default to kms as that's 'the' standard unit)

Then i'm happy to push out a gem release. I just hate pushing out gem releases which fixes 1 issue, but for others will cause a new issue.

**Comment by dmastylo** [comment]
@mnoack Totally understandable!

**Comment by tagliala** [comment]
@mnoack sorry I switched to another library
